engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -103,6 +103,8 @@
     private Context context;
     private int worldNumber;
     private String selectedWorld = "";
+    public int indexOfWorld;
+    public WorldSetupWrapper customWorld;
```

Comment:
Maybe a better name here - is this meant to reflect a custom _name_ for a given world? All worlds are sort of custom themselves.

Reply:
**copyOfSelectedWorld**  is it ok ?